rtvi-basic-demo/src/main/java/ai/rtvi/client/basicdemo/VoiceClientManager.kt

```diff
@@ -95,12 +62,52 @@ class VoiceClientManager(private val context: Context) {
         errors.add(Error(it.description))
     }
 
-    fun start(baseUrl: String) {
+    fun start(
+        baseUrl: String,
+        apiKey: String?,
+    ) {
 
         if (client.value != null) {
             return
         }
 
+        val options = VoiceClientOptions(
+            services = listOf(
+                ServiceRegistration("tts", "cartesia"),
+                ServiceRegistration("llm", "together"),
+            ),
+            config = listOf(
+                ServiceConfig(
+                    "tts", listOf(
+                        Option("voice", "79a125e8-cd45-4c13-8a67-188112f4dd22")
+                    )
+                ),
+                ServiceConfig(
+                    "llm", listOf(
+                        Option("model", "meta-llama/Meta-Llama-3.1-8B-Instruct-Turbo"),
+                        Option(
+                            "initial_messages", Value.Array(
+                                Value.Object(
+                                    "role" to Value.Str("system"),
+                                    "content" to Value.Str("You are a helpful voice assistant.")
+                                )
+                            )
+                        )
+                    )
+                )
+            ),
+            // Note: For security reasons, don't include your API key in a production
+            // client app. See: https://docs.dailybots.ai/architecture
+            customHeaders = apiKey
+                ?.takeUnless { it.isEmpty() }
+                ?.let { listOf("Authorization" to "Bearer $it") }
+                ?: emptyList(),
+            customBodyParams = listOf(
+                "bot_profile" to Value.Str("voice_2024_08"),
+                "max_duration" to Value.Number(680.0)
```

Comment:
In the future it will probably be nice we add that into the preferences, so they can try the different bot profiles.
I had a task for that on iOS. 😅